[CS2103T-T12-1] +Work

seahlynn opened the pull request:
@ArunBeCoding
@elsakoh
@gabrielseow
@ambhinav

SimNee commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,15 +299,22 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+|`* * *` |project leader |add tasks |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |project leader |change task status | ger reminder of the progress of each task
```
Small spelling error here

iskandarzulkarnaien commented on docs/DeveloperGuide.adoc:
```diff
@@ -315,33 +324,333 @@ _{More to be added}_
-[none]
-* 2a. The list is empty.
+2. The user does not specify name
+3. +work shows an error message
```
Numbering format is wrong. Should start from the step number where the extension occurs. Refer to [this](https://nus-cs2103-ay1920s1.github.io/website/schedule/week7/topics.html).

iskandarzulkarnaien commented on docs/DeveloperGuide.adoc:
```diff
@@ -315,33 +324,333 @@ _{More to be added}_
+2. The list is empty
 +
-Use case resumes at step 2.
+Use case ends
```
I think it would be better to have the system feedback to the user that the list is empty instead of simply terminating

SimNee commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,15 +299,22 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+|`* * *` |project leader |change task status | ger reminder of the progress of each task
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |project leader |sync team members schedules |Find a time slot where the maximum number of people, if not all, can attend
```
The capitalisation should be consistent. "find" is capitalise but before that is not

iskandarzulkarnaien commented on docs/DeveloperGuide.adoc:
```diff
@@ -315,33 +324,333 @@ _{More to be added}_
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. User requests to add a team member
+2. User specifies the name of the team member
```
This seems to be that at step 1, the user types in the command, then +Work waits for the user to type in the name of the team member. May I suggest adding a step in between where `+Work notifies user to key in the name of team member`

iskandarzulkarnaien replied:
This applies for your other use cases too

iskandarzulkarnaien replied:
Also, before use case ends, it may be good to notify the user that their action was successful.

SimNee commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,15 +299,22 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+|`* *` |Project leader with tasks that require inventory|assign tasks inventory or materials that are needed |Download an inventory report whenever required
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *`|Project leader with frequent meetings|Create a new meeting with recommended time and desired location|View my next meeting in the home page
```
Probably can split the function and benefit into 2 user stories since it does not seems to match.
As a project leader, I want to create meeting so that I can secure the meeting slot at my desired time and place.
As a project leader, I want to view my next meeting in the homepage so that I know my schedule at a glance.

iskandarzulkarnaien commented on docs/DeveloperGuide.adoc:
```diff
@@ -315,33 +324,333 @@ _{More to be added}_
+*Main Success Scenario*
+
+1. User requests to list team members
+2. +work displays list of team members
```
Seems like your app name is inconsistent in its capitalization. Some places have `+work` and others have `+Work`. Best to make it consistent.

SimNee commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,15 +299,22 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`*`|Project leader with a long-term project with ad-hoc members|Change members working on a task|Reassign tasks to incoming members
+
+|`*`|Project leader operating keeping track of purchases|Tag the inventory purchase to the member who bought it|Produce an accurate claims report at the end of the project
```
"Project leader operating keeping track of purchases" I don't seem to understand this. Could you rephrase it in a better way?

iskandarzulkarnaien commented on docs/DeveloperGuide.adoc:
```diff
@@ -315,33 +324,333 @@ _{More to be added}_
+
+*Main Success Scenario*
+
+1. User requests to mark a task as ‘doing
```
Missing close inverted comma

SimNee commented on docs/DeveloperGuide.adoc:
```diff
@@ -273,19 +273,21 @@ Refer to the guide <<Testing#, here>>.
+*Value proposition*: manage tasks assigned to project mates, finding common time slots and keep track of inventory faster than GUI apps.
 
 [appendix]
 == User Stories
```
Generally well structured and have all 3 parts present

iskandarzulkarnaien commented on docs/DeveloperGuide.adoc:
```diff
@@ -315,33 +324,333 @@ _{More to be added}_
+2. User specifies the task ID
+3. +work updates the dashboard
++
+Use case ends
```
Seems to be duplicated by mistake

SimNee commented on docs/DeveloperGuide.adoc:
```diff
@@ -297,15 +299,22 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+|`*`|Project leader operating keeping track of purchases|Tag the inventory purchase to the member who bought it|Produce an accurate claims report at the end of the project
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
 |=======================================================================
```
Did not really talk about the other type of users for this project. For example, project leader can assign tasks but did not talk about if the teammates can see the tasks

iskandarzulkarnaien commented on docs/DeveloperGuide.adoc:
```diff
@@ -315,33 +324,333 @@ _{More to be added}_
+
+1. User adds timetable of team mates
+2. User requests to generate availability timings of team members
+3. +work displays list of timings where the most number of team members are available
```
Same formatting issue here with extensions

SimNee commented on docs/DeveloperGuide.adoc:
```diff
@@ -315,33 +324,333 @@ _{More to be added}_
+Use case resumes at step 2
 
 [appendix]
 == Non Functional Requirements
```
1. Are the NFRs are really Non-Functional?
- Yes because it is not related to any function. They are function requirements related to performance, user-friendliness. 

2. Is each NFR well-defined (i.e., possible to decide when it has been met)?

- NFR 1 is a well-defined NFR, but very generic
- NFR 2: it is ambiguous as to what ‘noticeable sluggishness’ means. How many seconds/milliseconds of lag would it be before we notice the sluggishness?
- NFR 3: it is a measurable NFR. Since the wording is ‘accomplish most of the tasks faster using commands’, are there any specific tasks where it would take longer to accomplish?
- Other feedback: the NFRs are quite generic, and not specific to the application. 

3. Is each NFR reasonably achievable?
- NFRs are reasonably achievable

4. Are more relevant NFRs have been left out?
- Left out NFRs related to security. A possible NFR would be: only team leader can change the allocation of tasks to team members.

iskandarzulkarnaien reviewed (commented):
Good work in general! However there are formatting issues with use cases. Please refer to [this](https://nus-cs2103-ay1920s1.github.io/website/schedule/week7/topics.html)

weikiat97 commented on docs/DeveloperGuide.adoc:
```diff
@@ -64,9 +63,9 @@ image::LogicClassDiagram.png[]
 [discrete]
```
Diagrams do not have updated class name (it is still named as AddressBookParser). I think the ProjectDashBoardParser should be connected to the XYZCommandParser in this case.

weikiat97 commented on docs/DeveloperGuide.adoc:
```diff
@@ -98,9 +115,9 @@ image::LogicClassDiagram.png[]
 *API* :
```
The UI Component diagram is a bit difficult to understand. What are the functionalities of the UserView classes (Navigator / Controller / Main / Update)? Why is the UserViewMain connected to the Logic?

weikiat97 replied:
While the implementation may be unclear to us, the puml diagram is actually quite well done.

weikiat97 commented on docs/DeveloperGuide.adoc:
```diff
@@ -109,33 +126,33 @@ Given below is the Sequence Diagram for interactions within the `Logic` componen
 
 .Structure of the Model Component
-image::ModelClassDiagram.png[]
+image::ModelClassDiagramNew.png[]
```
Could consider putting the name/price/taskStatus under the attributes of the main classes.
Tag is an association, not a composition. Might want to change the notation for the tag.

weikiat97 commented on docs/DeveloperGuide.adoc:
```diff
@@ -109,33 +126,33 @@ Given below is the Sequence Diagram for interactions within the `Logic` componen
 
 [NOTE]
-As a more OOP model, we can store a `Tag` list in `Address Book`, which `Person` can reference. This would allow `Address Book` to only require one `Tag` object per unique `Tag`, instead of each `Person` needing their own `Tag` object. An example of how such a model may look like is given below. +
+As a more OOP model, we can store a `Tag` list in `Address Book`, which `Member` or 'Task' can reference. This would allow `Address Book` to only require one `Tag` object per unique `Tag`, instead of each `Member` or 'Task' needing their own `Tag` object. An example of how such a model may look like is given below. +
```
Don't forget to change the address books to your application! :)

weikiat97 commented on docs/DeveloperGuide.adoc:
```diff
@@ -153,7 +170,184 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
+Step 1. The user launches the application. The `ProjectCalendar` will be initialised based on the
+saved `ProjectCalendar`.
+
+Step 2. The user imports members' calendars by executing `impport-calendar`.
```
import (spelling error)

ambhinav committed:
Merge pull request #157 from ambhinav/abhinav-settings-feature

Fixed find command bug

ambhinav committed:
update some command example usage

seahlynn committed:
Updated to fix find member

seahlynn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

# Conflicts:
#	src/main/java/seedu/address/ui/UserViewNavigator.java

seahlynn committed:
Fixed find task UI

seahlynn committed:
Merge pull request #158 from seahlynn/master

Fixed find task UI

ambhinav committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main into abhinav-settings-feature

ambhinav committed:
fix app shutdown bug

ambhinav committed:
Merge pull request #159 from ambhinav/abhinav-settings-feature

Fixed application exit bug

ArunBeCoding committed:
UG1

ArunBeCoding committed:
Merge branch 'UG'

ArunBeCoding committed:
Merge pull request #161 from ArunBeCoding/master

UG-Inventory

ambhinav committed:
added unit tests for settings commands

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

seahlynn committed:
Updated UG

Commit:
removed instances of addressbook in packages

elsakoh committed:
Merge pull request #162 from elsakoh/master

removed references to addressbook

seahlynn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

ambhinav committed:
fix conflicts and merge

ambhinav committed:
fix package name change

ambhinav committed:
fixed failing tests caused by package rename

ambhinav committed:
optimise imports for project

ambhinav committed:
rename-refactor some task commands

ambhinav committed:
added unit tests for settings in model

ambhinav committed:
added settings command tests in parser

seahlynn committed:
Updated for task-stats

seahlynn committed:
Merge pull request #163 from seahlynn/master

Updated for task-stats

ambhinav committed:
added settings tests for storage

ArunBeCoding committed:
DG update

ArunBeCoding committed:
BugFixes-Inv

ArunBeCoding committed:
merge-DG updates

ArunBeCoding committed:
Merge pull request #164 from ArunBeCoding/master

BugFixes-PDF

ArunBeCoding committed:
MultiLine&UG-Refined

ArunBeCoding committed:
Merge pull request #165 from ArunBeCoding/master

MultiLine&UG-Refined

gabrielseow committed:
checkstyle changes

gabrielseow committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main into branch-add-sample-data

ambhinav committed:
Changed behaviour of setDeadline and editTask

Now setDeadline can only take in tasks without a deadline and set a new
one. EditTask can only take in tasks with a deadline and edit or remove
their deadline. Also wrote tests for setDeadline.

ambhinav committed:
fix conflicts and merge

ambhinav committed:
refactor to optimise imports

ambhinav committed:
change dateTime parsing to allow only future dates

ambhinav committed:
update dateTime util tests

ambhinav committed:
fix EOF errors

ambhinav committed:
fix gradlew.bat

ambhinav committed:
change line seperator

ambhinav committed:
Merge pull request #166 from ambhinav/abhinav-settings-feature

Wrote tests for settings and deadline-related commands

gabrielseow committed:
integrated undo/redo functionality and removed ListTaskByMember

seahlynn committed:
Updated DG

seahlynn committed:
Sequene diagrams

seahlynn committed:
Added class diagram for DG

ambhinav committed:
update UG and PPP

gabrielseow committed:
standardised Meeting List fxml according to main app

gabrielseow committed:
creating addMeetingCommandTest

gabrielseow committed:
dealing with merge conflicts from team master

gabrielseow committed:
add meeting command test almost complete

gabrielseow committed:
fixed errors with addMeetingCommandTest

gabrielseow committed:
fixed test errors

gabrielseow committed:
Merge pull request #167 from gabrielseow/master

Changed MeetingList fxml and added Meeting tests

Commit:
updates to autocorrect drop down styling to make sure it is not invisible in light theme

gabrielseow committed:
added checks for invaid start time and end time

gabrielseow committed:
added update support for deleted Calendars

seahlynn committed:
Updated DG

seahlynn committed:
Updated DG

gabrielseow committed:
fixed fxml display for members under a meeting

seahlynn committed:
Updated DG

gabrielseow committed:
modified duplicateCalendarException to check for membername only

seahlynn committed:
Updated DG

seahlynn committed:
Edited PPP

ambhinav committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main into abhinav-UG-DG

gabrielseow committed:
added test cases for AddMeetingCommandTest and removed check if deadline has passed in DateTimeUtil

ambhinav committed:
update UG to include latest pictures

gabrielseow committed:
added test cases for FindMeetingTimeCommandTest

gabrielseow committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

gabrielseow committed:
Merge pull request #169 from gabrielseow/master

Added tests for FindMeetingTimeCommand and AddMeetingCommand

ambhinav committed:
updated UG to include edit-task and pics

seahlynn committed:
Updated PPP

seahlynn committed:
Updated PPP

seahlynn committed:
Updated PPP

seahlynn committed:
Updated PPP

seahlynn committed:
Updated PPP (trying to shorten it)

seahlynn committed:
Updated PPP

seahlynn committed:
Updated PPP

seahlynn committed:
Updated PPP

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

Commit:
travis fix

elsakoh committed:
Merge pull request #168 from elsakoh/master

autocorrect dropdown styling

seahlynn committed:
Added test for edit member command

ArunBeCoding committed:
TestCases

ArunBeCoding committed:
Merge branch 'TestCases'

ArunBeCoding committed:
Merge pull request #170 from ArunBeCoding/master

InventoryTests

seahlynn committed:
Added test for addmembercommand parser

seahlynn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

# Conflicts:
#	docs/DeveloperGuide.adoc
#	src/test/java/seedu/pluswork/logic/commands/CommandTestUtil.java

ArunBeCoding committed:
Update README.adoc

ArunBeCoding committed:
Update README.adoc

ArunBeCoding committed:
Update README.adoc

ArunBeCoding committed:
UG-updatePics

ArunBeCoding committed:
Merge pull request #171 from ArunBeCoding/master

UG-updatePics

ambhinav committed:
update DG dashbaord feature

seahlynn committed:
Edited PPP

seahlynn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

gabrielseow committed:
added acknowledgements for iCal4j external library

gabrielseow committed:
merged from team master and fixed merge conflicts

ambhinav committed:
updated DG to organise UI component

Also deleted unused views such as Inventory and TaskWithMember cards

seahlynn committed:
Fixed checkstyle

seahlynn committed:
Fixed checkstyle

seahlynn committed:
Fixed checkstyle for diagrams

seahlynn committed:
Merge pull request #172 from seahlynn/master

Fixed Checkstyle, added test for edit member command

ambhinav committed:
fix conflicts and merge with #172

ambhinav committed:
Merge pull request #173 from ambhinav/abhinav-UG-DG

Updates to UG and DG for Ui and settings

gabrielseow committed:
added test cases for FindMeetingTimeCommand and AddCalendarCommand

gabrielseow committed:
dealt with merge conflicts from team master

gabrielseow committed:
fixed some import errors from merge conflicts

gabrielseow committed:
Merge pull request #174 from gabrielseow/master

Added test cases for FindMeetingTimeCommand and AddCalendarCommand

ambhinav committed:
update PPP intro and contributions

ambhinav committed:
update UG settings feature

ambhinav committed:
update PPP and DG to improve content

ambhinav committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main into abhinav-UG-DG

ambhinav committed:
Merge pull request #175 from ambhinav/abhinav-UG-DG

Update my UG, DG and PPP

seahlynn committed:
Updated PPP

seahlynn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

ambhinav committed:
finalise PPP attempt 1

ambhinav committed:
change UI to Ui in PPP

seahlynn committed:
Updated pictures for UG (interlaced image)

ambhinav committed:
fix PPP spelling errors

ambhinav committed:
Merge pull request #176 from ambhinav/abhinav-UG-DG

Finalise PPP attempt 1

gabrielseow committed:
edited UG

seahlynn committed:
Updated UG, DG and PPP

seahlynn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

ambhinav committed:
finalise PPP v2

ambhinav committed:
Merge pull request #177 from ambhinav/abhinav-UG-DG

finalise PPP v2

gabrielseow committed:
added displayDateRange method in DateTimeUtil

seahlynn committed:
Updated PPP

seahlynn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

ambhinav committed:
resize images for UG/DG submission

ambhinav committed:
update AboutUs.adoc

ambhinav committed:
fix addTaskIntegration test

seahlynn committed:
Updated UG, DG, PPP

seahlynn committed:
Merge pull request #178 from seahlynn/master

Updated UG, DG, PPP

seahlynn committed:
HTML vers of PPP

seahlynn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

seahlynn committed:
HTML vers of Abhi's PPP

seahlynn committed:
Update AboutUs.adoc

seahlynn committed:
Fixed checkstyle

seahlynn committed:
Merge pull request #179 from seahlynn/master

HTML vers of PPP

seahlynn committed:
Updated DG to include manual testing

ambhinav committed:
update DG manual testing for settings, task and db

ambhinav committed:
fix conflicts and merge

ambhinav committed:
update DG links

ambhinav committed:
update README.md

ambhinav committed:
Merge pull request #181 from ambhinav/abhinav-UG-DG

Updates to DG and README

Commit:
support for using shift key for autocomplete

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

elsakoh committed:
Merge pull request #182 from elsakoh/master

support for using shift key for autocomplete

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

Commit:
ppp

ArunBeCoding committed:
arun-PPP updates

ArunBeCoding committed:
Merge pull request #183 from ArunBeCoding/master

arun-PPP updates

seahlynn committed:
Fixed task-stats

seahlynn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

# Conflicts:
#	docs/DeveloperGuide.adoc

gabrielseow committed:
fixed error for DeleteMeetingCommand

seahlynn committed:
Merge pull request #180 from seahlynn/master

Updated DG to include manual testing

seahlynn committed:
Made sure task-stats works

gabrielseow committed:
update UG

gabrielseow committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

seahlynn committed:
Fixed test

seahlynn committed:
Merge pull request #184 from seahlynn/master

Made sure task-stats works

ambhinav committed:
fix add task in UG

gabrielseow committed:
finalised UG update

Commit:
UG, DG, PPP and bug fix for autocomplete

ambhinav committed:
fix checkstyle

ambhinav committed:
reformat code to optimise imports

ambhinav committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main into abhinav-UG-DG

ArunBeCoding committed:
BugFix

ArunBeCoding committed:
Merge pull request #186 from ArunBeCoding/master

BugFix

gabrielseow committed:
added half test cases for DeleteCalendarCommandTest

gabrielseow committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

gabrielseow committed:
changed java gradle version to java11

gabrielseow committed:
Merge pull request #185 from gabrielseow/master

Updated UG and fixed major bug in DeleteMeetingCommand

ambhinav committed:
renamed editcommandparser to include task

ambhinav committed:
add comments to make code clearer

ambhinav committed:
organise logic classes

Commit:
ppp pdf and minor bug fix for autocomplete

Commit:
fix merge conflict

Commit:
travis fix

ambhinav committed:
organise classes in logic test folder

ambhinav committed:
reformat code to optimise imports

ambhinav committed:
fix conflicts and merge with #185

elsakoh committed:
Merge pull request #187 from elsakoh/elsakoh-UG-DG

Elsakoh-ug-dg

ambhinav committed:
fix tests

ambhinav committed:
fix conflicts and merge

ambhinav committed:
update UG faq

ambhinav committed:
Merge pull request #188 from ambhinav/abhinav-UG-DG

Clean up code in preparation for v1.4

ArunBeCoding committed:
Delete Ui.png

ArunBeCoding committed:
Add files via upload

ArunBeCoding committed:
Delete Ui.png

ArunBeCoding committed:
no message

ArunBeCoding committed:
Merge pull request #189 from ArunBeCoding/master

Ui.png

ArunBeCoding committed:
Update arunkumarr.html

Commit:
minor bug fix for autocomplete prefix usage for edit-inv

ambhinav committed:
update UG formatting and content

Commit:
travis checks

elsakoh committed:
Merge pull request #190 from elsakoh/master

minor bug fix for autocomplete prefix usage for edit-inv

ambhinav committed:
update some class diagrams in DG

ambhinav committed:
fix conflicts and merge

ambhinav committed:
Merge pull request #191 from ambhinav/abhinav-UG-DG

Updates to UG and DG formatting and diagrams

ArunBeCoding committed:
Update README.adoc

Commit:
undo to lower case

ambhinav committed:
refactor user view update

elsakoh committed:
Merge pull request #192 from elsakoh/master

undo to lower case

ambhinav committed:
fix checkStyle

ambhinav committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main into abhinav-UG-DG

ambhinav committed:
Merge pull request #193 from ambhinav/abhinav-UG-DG

Refactor user view update

Commit:
fix for member index

ambhinav committed:
improve task command res messages

ambhinav committed:
fix upcoming deadlines bug

ambhinav committed:
Merge pull request #194 from ambhinav/master

Add sample tasks and fix upcoming deadlines bug

Commit:
updates to ppp

Commit:
travis fix

elsakoh committed:
Merge pull request #195 from elsakoh/master

update to elsakoh-ppp

seahlynn committed:
Updated DG

seahlynn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

gabrielseow committed:
updated DG

gabrielseow committed:
merged from team master

gabrielseow committed:
Merge pull request #196 from gabrielseow/master

Updated UG and DG

seahlynn committed:
Added more sample members

seahlynn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

# Conflicts:
#	docs/DeveloperGuide.adoc

seahlynn committed:
Merge pull request #197 from seahlynn/master

Added more sample memebrs

Commit:
change control to right arrow and ppp updates

ambhinav committed:
giving credit

ambhinav committed:
removed a line

ambhinav committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

Commit:
ppp update

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

Commit:
travis eof fix

elsakoh committed:
Merge pull request #198 from elsakoh/master

fix paste to command box not working

ambhinav committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

seahlynn committed:
Cleaning up ug and dg

seahlynn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

ambhinav committed:
Merge pull request #199 from ambhinav/master

Giving credit

Commit:
ppp pdf

elsakoh committed:
Merge pull request #200 from elsakoh/master

elsakoh-ppp-pdf

ambhinav committed:
fix tests

seahlynn committed:
Updated UG and DG

ambhinav committed:
Merge pull request #201 from ambhinav/master

Finalise for v1.4

seahlynn committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-T12-1/main

# Conflicts:
#	docs/team/elsakoh.pdf

seahlynn committed:
Merge pull request #202 from seahlynn/master

Final UG and DG

gabrielseow committed:
added ppp

gabrielseow committed:
Merge pull request #203 from gabrielseow/master

added ppp

ArunBeCoding committed:
Rename arunkumarr.adoc to ArunBeCoding.adoc

ArunBeCoding committed:
Delete arunkumarr.pdf

ArunBeCoding committed:
Add files via upload
